server/src/api/roster/roster-entity.ts

```diff
@@ -57,6 +57,11 @@ export abstract class RosterEntity extends BaseEntity {
   })
   lastName!: string;
 
+  @Column({
+    length: 20, default: 'Missing',
```

Comment:
What do you think about just making this column nullable, and letting null be the default? I'd rather leave the textual representation of a missing phone number up to the frontend, and making missing numbers null will let us check for non-existent phone numbers with `!phoneNumber`.

Reply:
Definitely a better!

Reply:
done